src/views/preview/preview.scss

```diff
@@ -28,6 +28,19 @@ $medium-and-small: "screen and (max-width : #{$tablet}-1)";
 
 .preview {
 
+    .inner {
+        @media #{$medium-and-small} {
+            margin: 0 auto;
+            max-width: 90%;
```

Comment:
I'm curious about `90%` — I think elsewhere we use a static value or `100%`, was that approach not appropriate here?

Reply:
In order to fit the in between sizes I didn't want to use a static value. And I wanted to leave space so I used 90%. However, now that you mention it, I could also try something else... let me check...

Reply:
I guess I could use padding. But I think I would want to get this pull request in after the other changes went in. It is hard for me to test this way if things will work.